apps/web/package.json

```diff
@@ -32,7 +32,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.2.0",
     "@babel/plugin-proposal-optional-chaining": "^7.2.0",
-    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-flow": "^7.0.0-rc.1",
```

Comment:
why are we moving from `7.0.0` -> `7.0.0-rc.1`

also this dependency should only be specified once. Please move it to peerDependency in on of the package.json files.